demo/kserve/scripts/test/deploy-model.sh

```diff
@@ -4,7 +4,24 @@ set -o nounset
 set -o errtrace
 # set -x   #Uncomment this to debug script.
 
+# Usage: a single! arg: "http" or "grpc" - the protocol to be used 
+
+# Check if a single argument is passed
+if [ "$#" -ne 1 ]; then
+    echo "Error: exactly one argument is required: either 'http' or 'grpc'"
+    exit 1
+fi
+
+# Check if the argument is either "http" or "grpc"
+if [ "$1" = "http" ] || [ "$1" = "grpc" ]; then
+    INF_PROTO=$1
+else
+    echo "Error: Argument must be either 'http' or 'grpc'."
+    exit 1
+fi
+
 source "$(dirname "$(realpath "$0")")/../env.sh"
+export TEST_NS=${TEST_NS}"-$INF_PROTO"
```

Comment:
Not sure we want to restrict each to a specific NS, you could have two KServe ServingRuntimes in a single NS, one for each protocol

Reply:
Done,  we have now  a single NS